examples/01_plotting/plot_demo_glass_brain.py

```diff
@@ -26,4 +26,9 @@
     localizer_tmap_filename, title='plot_glass_brain',
     black_bg=True, display_mode='xz', threshold=3)
 
+###############################################################################
+# Hemispheric saggital cuts
+plotting.plot_glass_brain(
+    localizer_tmap_filename, display_mode='lyrz', threshold=3)
```

Comment:
It would be easy to catch quickly using title something like plot_glass_brain with display_mode='lyrz'